ledger/ledger-common/src/main/kotlin/net/corda/v5/ledger/common/transactions/SignedTransaction.kt

```diff
@@ -0,0 +1,33 @@
+package net.corda.v5.ledger.common.transactions
+
+import net.corda.v5.application.crypto.DigitalSignatureAndMetadata
+import net.corda.v5.base.annotations.CordaSerializable
+import net.corda.v5.base.annotations.DoNotImplement
+import java.security.PublicKey
+
+@DoNotImplement
+@CordaSerializable
+interface SignedTransaction {
+    val wireTransaction: WireTransaction
+    val ledgerTransaction: LedgerTransaction //TODO(do we need this? Or rather delegate some properties through?)
```

Comment:
We should not have a property for ledgerTransaction as the conversion process is complicated and contextual and this might easily get serialized by mistake. We can have a function that returns a LedgerTransaction on demand and makes it clear that it is not for serialization

Reply:
Thanks. Done.